db/models.py

```diff
@@ -47,8 +47,17 @@ def validate_value(cls, value):
             return value
         if isinstance(value, str) and value in ("any", "untagged"):
             return value
+        if isinstance(value, str):
+            tags = value.split("-")
```

Comment:
@ajoaoff, @italovalcy requested a bit more flexibility, including non contiguous ranges, in the [requirements discussion](https://github.com/kytos-ng/kytos/issues/318#issuecomment-1470419792), it was mentioned something like `{"tag": {"value": "1-100,200-300,400", "tag_type": 1}`, you'll probably need a regex or multiple `split`s.

Good thing that `openapi` also supports (regex) string `pattern` validation, so for the request it should also be able to validate upfront.

Reply:
Right, I had forgotten about multiple ranges.

Reply:
I got to this regex `^\d+(-\d+)?(,\d+(-\d+)?)*$`. It works to match the possible values, but won’t enforce that, for `a-b`, `a <= b`.

Reply:
This regex looks good to me @ajoaoff. Ensuring that they're ascending is pivotal at the boundaries though. Getting this validation at the `pydantic` level whenever `evc.sync()` would be nice, but still `UNI` creation will take place before that. So, I suppose we can go as it is, but then you'd need to cover that part on `UNI.is_valid` wdyt? Although from what I followed `_validate` is only being called when an `EVC` is instantiated so updating we might also need to ensure that it also validates that part. 

Also, in the original linked comment I mentioned an idea there about a potential `UNIRange`, let me know how you're planning to cover that part or if you see any other way that can be simpler to maintain while ensuring no race conditions.

Reply:
I think we have a design choice to make here. We can either:
- store the ranges in the format inputted by the user; this way we can use the value as is in API responses but need to calculate `vlan/mask` at deploy time;
- calculate all `vlan/mask` when creating/updating; this way we can deploy as it is but need to convert back in API responses.

Reply:
Right @ajoaoff. I think the first approach is fine as far as from the DB point of view. My major concern at the moment though is also making sure show the `UNI` and potential `UNIRange` will evolve since at runtime before actually storing the EVCs we need that part structured and covered, since it needs upfront to ensure without race conditions that the vlans are available, and then after that cover the rest pushing the flow mods based on the minimum vlan/mask pairs required. 

We could potentially land this PR here, since the openapi.yml validations `#/components/schemas/Tag`'s `value` string is still a `enum` there, but I'd rather only land knowing that the rest of the related changes will also land on `2023.1`, to ensure that the DB won't give margin to store things that can't still completely deal with. Do you think you can still contribute to this feature for `2023.1`? If you won't be able that's completely understandable as you've explained, and if that's the case then we'll need to reassess if another core team member can finish the other parts. 

Let me know. Thanks, Antonio.